spec/index.html

```diff
@@ -805,23 +805,25 @@ <h3>Reified Triples</h3>
     <p>A <a>reified triple</a> is represented using the
       <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
       starting with <a href="#cp-double-lt"><code>&lt;&lt;</code></a>,
-      followed by <a href="#grammar-production-subject"><code>subject</code></a>,
+      followed by <a href="#grammar-production-ttSubject"><code>ttSubject</code></a>,
       <a href="#grammar-production-predicate"><code>predicate</code></a>, and
       <a href="#grammar-production-ttObject"><code>ttObject</code></a>,
-      optionally follwed by a <a href="#grammar-production-reifier"><code>reifier</code></a>
-      (<a href="#grammar-production-IRIREF"><code>IRIREF</code></a>
-      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>),
+      optionally followed by a <a href="#grammar-production-reifier"><code>reifier</code></a>,
+      composed of a <a href="#cp-tilda"><code title="tilda">~</code></a>
+      followed by an <a href="#grammar-production-iri"><code>iri</code></a>
```

Comment:
It's tricky to know whether this (and other instances) should be `IRI` or `<code>iri</code>`. I've made best-guesses in this PR.
```suggestion
      followed by an <a href="#grammar-production-iri">IRI</a>
```

Reply:
Note that the [IRI ](https://www.w3.org/TR/rdf12-concepts/#dfn-iri) term defined in RDF Concepts is distinct from the <a href="#grammar-production-iri"><code>iri</code></a> BNF production.